src/appengine/server.py

```diff
@@ -162,6 +163,7 @@ def get(self, _):
     ('/corpora', corpora.Handler),
     ('/corpora/create', corpora.CreateHandler),
     ('/corpora/delete', corpora.DeleteHandler),
+    ('/csp-reports', csp_reports.CspReportHandler),
```

Comment:
should this be named report-csp-failure ? since this is not a page to view csp-reports but to report a particular failure ?